src/tools/rados/rados.cc

```diff
@@ -67,11 +67,6 @@ void usage(ostream& out)
 "usage: rados [options] [commands]\n"
 "POOL COMMANDS\n"
 "   lspools                          list pools\n"
-"   mkpool <pool-name> [123[ 4]]     create pool <pool-name>'\n"
-"                                    [with auid 123[and using crush rule 4]]\n"
-"   cppool <pool-name> <dest-pool>   copy content of a pool\n"
```

Comment:
I don't like the cppool command, but it does see some use in the community and at a quick skim of the mailing list I'm not sure all that use is broken. :/ I know you asked about it on the auid dev list thread, but we may need to discuss that more broadly.